packages/@ember/-internals/glimmer/lib/utils/iterable.ts

```diff
@@ -127,7 +135,22 @@ class ObjectIterator extends BoundedIterator {
     } else {
       let values: Opaque[] = [];
       for (let i = 0; i < length; i++) {
-        values.push(get(obj, keys[i]));
+        let value: any;
+        let key = keys[i];
+
+        value = obj[key];
```

Comment:
This breaks when `obj` is a proxy, is that okay?

In general, if the solution requires avoiding `get` semantics, I think that is not okay. However, maybe in this case it is fine since we already did `Object.keys` above so it won't trigger `unknownProperty` anyway.

Reply:
Correct, I started writing the code to do `unknownProperty` and realized that by definition, it could never run, which is why I omitted it.